Core/Src/main.c

```diff
@@ -65,7 +71,8 @@ void SystemClock_Config(void);
 int main(void)
 {
   /* USER CODE BEGIN 1 */
-
+	char spi_buf_rx[10];
```

Comment:
why the received buffer has a size of 10?

Reply:
I thought it was a size 10 because there are 10 bits of information being transmitted.... @HardyYu